rosidl_runtime_py/rosidl_runtime_py/convert.py

```diff
@@ -59,8 +59,12 @@ def __represent_ordereddict(dumper, data):
     return yaml.nodes.MappingNode(u'tag:yaml.org,2002:map', items)
 
 
-def message_to_yaml(msg: Any, truncate_length: int = None,
-                    no_arr: bool = False, no_str: bool = False) -> str:
+def message_to_yaml(
+        msg: Any,
+        *,
+        truncate_length: int = None,
+        no_arr: bool = False,
+        no_str: bool = False) -> str:
```

Comment:
Nitpick: use single level of indentation here and wrap again before the closing parenthesis.

Same below.

Reply:
Done: 1477de2

Maybe we should consider adding a note to our Python style guide. [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation) is happy with the previous two styles.

Reply:
See https://github.com/ros2/ros2_documentation/pull/319